compose.yaml

```diff
@@ -20,43 +10,46 @@ services:
   devnet:
     container_name: devnet
     build:
-      context: ./rollups-node/test/devnet
+      context: ./ci/devnet
       dockerfile: Dockerfile
     ports:
       - 8545:8545
-    volumes:
-      - ./rollups-node/deployment.json:/usr/share/devnet/deployment.json
-      - ./rollups-node/anvil_state.json:/usr/share/devnet/anvil_state.json
   espresso-dev-node:
     image: ghcr.io/espressosystems/espresso-sequencer/espresso-dev-node:20241120-patch6
     environment:
       ESPRESSO_SEQUENCER_L1_PROVIDER: http://devnet:8545/
       ESPRESSO_SEQUENCER_API_PORT: 24000
       ESPRESSO_SEQUENCER_L1_POLLING_INTERVAL: 1s
       ESPRESSO_STATE_PROVER_UPDATE_INTERVAL: 1s
-      ESPRESSO_SEQUENCER_POSTGRES_HOST: db-espresso
+      ESPRESSO_SEQUENCER_POSTGRES_HOST: db
       ESPRESSO_SEQUENCER_POSTGRES_PORT: 5432
       ESPRESSO_SEQUENCER_POSTGRES_DATABASE: rollupsdb
```

Comment:
Should we use the same DB as Rollups? Espresso Dev Node's DB is a very separate thing
We could also simply not specify anything, and let Espresso Dev Node do as it pleases (e.g., if they decide not to use PostgreSQL, I'd rather we don't care)